README.md

```diff
@@ -12,62 +12,47 @@ class for each native type:
 * [`ShortPreference`][short-pref] for a `short` value
 * [`StringPreference`][string-pref] for a `String` value
 
-Each class comes with two constructors - one of them allows to ommit
-the default value.
-
-All classes declare the same methods: `get()`, `isSet()`, `set()` and `delete()`.
-Here are the method signatures as definied in the `StringPreference` class.
-
-```java
-public class StringPreference {
-    public String get() {  }
-    public boolean isSet() {  }
-    public void set(String value) {  }
-    public void delete() {  }
-}
-```
-
 ## Source
 
-Inspired by https://github.com/johnjohndoe/TypedPreferences - version 1.0
+Inspired by https://github.com/johnjohndoe/TypedPreferences - v2.1.0 version
 
+## Feature
+
+<img src="Screenshot/ezgif.com-gif-maker.gif" width="500">
 
 ## Dependency
 1. For using apprater module in sample app, include the source code and add the below dependencies in entry/build.gradle to generate hap/support.har.
 ```groovy
     dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar', '*.har'])
-    implementation project(path: ':library')
+    implementation project(path: ':typedpreferences'
+)
     testImplementation 'junit:junit:4.13'
```

Comment:
All the dependencies must be present within thin the dependency node. Please correct this